docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+services:
+  javaps:
+    build: .
+    image: wacodis/javaps:latest
+    env_file:
+      - ./docker/wacodis.env
+    ports:
+      - 8081:8080
+    deploy:
+      resources:
+        limits:
+          memory: 4G
+        reservations:
+          memory: 2G
+    environment:
+      SERVICE_SERVICE_URL: http://localhost:8081/wacodis-javaps/service
+      SERVICE_PROVIDER_INDIVIDUAL_NAME: Sebastian Drost
+      SERVICE_PROVIDER_POSITION_NAME: First Line Supporter (24/7)
```

Comment:
Seems discussable to me...

Reply:
If you deploy the service with this docker-compose template, its your own fault ;-)

Reply:
I won't, but others could ;-)